src/index.html

```diff
@@ -17,6 +17,10 @@
     />
   </head>
   <body>
-    <h1>Stop watch</h1>
+    <div class="stopwatch stopwatch--speed-up">
+      <div class="stopwatch__center"></div>
+      <div class="stopwatch__seconds-hand"></div>
+      <div class="stopwatch__minutes-hand"></div>
+    </div>
```

Comment:
Ensure that all BEM elements have the block class listed first, and then the modified class. For example, it should be `class="stopwatch stopwatch--speed-up"`.